public/app/screens/main/navigation/navigation.jsx

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var React = require('react'),
+  Link = require('react-router').Link,
+  StyleMixin = require('mixins/style-mixin');
+
+var Navigation = React.createClass({
+  mixins: [StyleMixin(require("./style.scss"))],
+  render: function () {
+    // TODO: Use mixins instead of CSS classes
+    // TODO: Pass number of elements to mixin instead of specifying class name
+    var classes = "cc_navigation icon-bar two-up " + this.props.orientation;
+    
+    var items = this.props.items.map(function (item, i) {
+      return (
+        <Link to={item.linkTo} key={i} className="cc_navigation__link item">
```

Comment:
Not defining the `key` property will however fire a warning onto the console. This is for good reason as the loop variable might not always be a good key (see the blog entry). This said I would prefer to always define a `key` explicitly so we never forget it.